manifests/agent/windows.pp

```diff
@@ -1,9 +1,14 @@
-class distelli::agent::windows (
-  $version = '3.66.33',
+# Install the Puppet Pipelines agent on Windows
+#
+# Do not use directly; use pipelines::agent.
+class pipelines::agent::windows (
   $tempdir = 'C:/Users/distelli/AppData/Local/Temp/1',
```

Comment:
Dangerous Default.  Do not hardcode this.

Reply:
Should be using a TMP or TEMP fact

In this case I know that that temp directory is used during RDP sessions, not local logons.